### ISSUE TITLE
iTunes 8 breaks application

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MziRintu/iphonebrowser#54]
```
What steps will reproduce the problem?
1. Install iTunes 8
2. Connect iPhone
3. Start iPhoneBrowser

What is the expected output? What do you see instead?
Application should start as per normal and show GUI.  Application fails to
open.

What version of the product are you using? On what operating system?
v1.70 on Windows XP SP2

Please provide any additional information below.
The bug is also apparent if you start iPhoneBrowser without the iPhone
connected but in this instance the GUI will be displayed until you connect
the iPhone and then it will close.
```

Original issue reported on code.google.com by `calvin.c...@gmail.com` on 10 Sep 2008 at 12:15

**Comment by GoogleCodeExporter** [comment]
```
Same crash happens with v1.52
```

Original comment by `chrischi...@gmail.com` on 10 Sep 2008 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.80
```

Original comment by `NetMage....@gmail.com` on 11 Sep 2008 at 6:08
- Changed state: **Duplicate**
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**
